File: test/test_main.cc
// Copyright 2017 The Fuchsia Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

#include "gtest/gtest.h"

// This is a generic GTEST entry-point to be reused by all unittest binaries
// under bluetooth.

int main(int argc, char** argv) {
  testing::InitGoogleTest(&argc, argv);
  return RUN_ALL_TESTS();
}

File: examples/diagnostics/inspect/dart/inspect_mod/lib/test_main.dart
// Copyright 2021 The Fuchsia Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

import 'package:flutter_driver/driver_extension.dart';

import 'main.dart' as entrypoint;

void main() {
  enableFlutterDriverExtension();
  entrypoint.main();
}
